Update accuracy claim in README description (#210)
fingerprintjs/fingerprint-pro-server-api-openapi: README.md

# Fingerprint Pro Server API OpenAPI Schema

> [Fingerprint](https://fingerprint.com) is a device intelligence platform offering industry-leading accuracy. Fingerprint [Server API](https://dev.fingerprint.com/reference/pro-server-api) allows you to retrieve visitor history or individual identification events in a server environment.

Reviewer: Server API has more capabilities now

```suggestion
[Fingerprint](https://fingerprint.com) is a device intelligence platform offering industry-leading accuracy. Fingerprint [Server API](https://dev.fingerprint.com/reference/pro-server-api) allows you to search, update, and delete identification events in a server environment.
```
